services/project-grafana-logging/8.0.2-d2iq.0/defaults/cm.yaml

```diff
@@ -9,7 +9,8 @@ data:
     priorityClassName: "dkp-critical-priority"
     # pinning grafana to 8.5 since it appears that v9+ is causing the kubernetes audit dashboard to crash
     image:
-      tag: 10.3.3
+      repository: ghcr.io/mesosphere/dkp-container-images/docker.io/grafana/grafana
+      tag: 10.3.3-d2iq.0
```

Comment:
I'm not sure why we are overriding the grafana image here, but I'm thinking maybe we remove the override and use the default deployed with the chart. But this can be done in a separate PR (with the grafana bump PRs)

Reply:
doing this in https://github.com/mesosphere/kommander-applications/pull/2416